iina/PrefCodecViewController.swift

```diff
@@ -110,6 +110,17 @@ class PrefCodecViewController: PreferenceViewController, PreferenceWindowEmbedda
     hwdecDescriptionTextField.stringValue = hwdec.localizedDescription
   }
 
+  // TODO: add "auto" option instead of 0 value
```

Comment:
Might want to support `auto`, but it should _not_ be a replacement for `0`. Setting the [--target-peak](https://mpv.io/manual/stable/#options-target-peak) option to `auto` has a specific meaning, documented in the mpv manual. Zero has been grabbed by IINA for its own implementation that attempts to actually detect the peak of the display. We'd want to add something like `IINA detected`. Something to distinguish from the mpv behavior.

Reply:
Good point. I hadn't put too much thought into it. So there are actually 3 special values apart from the normal number range. Might want to add additional UI for those in the future...

The TODOs have been removed :)